scene/2d/skeleton_2d.cpp

```diff
@@ -48,12 +48,14 @@ bool Bone2D::_set(const StringName &p_path, const Variant &p_value) {
 	} else if (path.begins_with("default_length")) {
 		set_length(p_value);
 	}
-
```

Comment:
This is enforced by format, unless I move the closing brackets into the `#ifdef`